### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by kkcroft71** [arkypita/LaserGRBL#1846] [General]
can i burn the same image on multile things at one time?

**Comment by khturnings** [comment]
You have provided very little info so no one will be able to answer this question correctly.

The best will be it depends.
Yes if your laser is powerful enough no if it is not.

What power laser are you using? What material?

**Reply by kkcroft71** [reply]
I have a atomstack a5 and its on thin thin wood i want to set up at least 10 to burn at the same time instead of having to burn 1 at a time. Ive saw videos of others doing it, but i can not figure out how.  I have already engraved a big bamboo wood cutting board and it looks great.
